### ISSUE TITLE
Add Virtual SAN health check 

### ISSUE THREAD

**Issue by cordesc** [alanrenouf/vCheck-vSphere#442]
Could you add a section that tests the health of a vsan?

**Comment by Sneddo** [comment]
What sort of thing are you looking for that the existing VSAN plugins don't provide?

**Comment by cordesc** [comment]
The plugins provide everything I need. I'm just looking for a way to get the current status returned via powerCli and run the ad-hoc tests.

Here are the steps I'd like to automate. In the end if you know of the cmdlets I can write the script. I just could not find them in the latest storage module of powercli 6 R3

1. Click the Home button > Choose Hosts and Clusters
2. Expand the datacenter
3. Select the cluster >  Select the Monitor tab > Select the Virtual SAN button
4. Select Physical Disks > Scroll the Disk Group window all the way to the right > Verify that all disks show Mounted in the Operational column and a Healthy status in the Virtual San Health status column
5. Select Virtual Disks > Confirm all disks show Compliant in the Compliance Status Column and Healthy in the Operational State column
6. Select Health > Click the Retest button > Confirm that all tests show as Passed in the Test Result column
7. Select Proactive Tests > Select the first test > Click the play button > Repeat for all the available tests > After the tests finish, all tests should show as Passed in the Last Run Result column.

**Comment by Sneddo** [comment]
OK, this is going to have to be added by someone that has access to a VSAN setup.

Get-VSANDisk and Get-VSANDiskGroup might provide some of this info, the rest may need to dig a little deeper.

**Comment by cordesc** [comment]
Yeah, those get the current status of the cluster. What I'm interested in is the ad-hoc execution of the health checks.
